test/Sentry.Maui.Tests/SentryMauiOptionsTests.cs

```diff
@@ -56,4 +58,40 @@ public void AttachScreenshots_Default()
         var options = new SentryMauiOptions();
         Assert.False(options.AttachScreenshot);
     }
+
+    [Fact]
+    public void BeforeCaptureScreenshot_Default()
+    {
+        // Arrange
+        var options = new SentryMauiOptions();
+        options.AttachScreenshot = true;
+
+        // Act
+        options.SetBeforeScreenshotCapture((@event, hint) =>
+        {
+            return true;
+        });
+
+        // Assert
+        Assert.NotNull(options.BeforeCaptureInternal);
+
+    }
+
+    [Fact]
+    public void BeforeCaptureScreenshot_Skip_Screenshot()
```

Comment:
These two tests both seem to do basically the same thing... which is to ensure that when you call SetBeforeScreenshotCapture the internal field is not null. 

I think it'd be better to have one test where you set a value (which is what you have) and another test where you don't set a value... and make sure you have the expected behaviour in that second scenario as well.